src/python/gem5/resources/resource.py

```diff
@@ -25,6 +25,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 from abc import ABCMeta
+import json
```

Comment:
Needed?

Reply:
The second commit removes the unneeded import, I will rebase everything into one commit after all requested changes are done.